### DISCUSSION TITLE
Chirp release v0.8.1

### DISCUSSION THREAD

**Discussion by TheDireHavok** [egzumer/uvk5-chirp-driver#25] [Q&A]
Hello, new to this Custom FW version of Chirp which was downloaded from this Repo. I have installed and using and running fine, however when selecting radioreference as a query source it doesn't open a signin window like the other 3 do and have no way to us it otherwise. Is this disabled in programming? Any help would be great.

**Comment by egzumer** [comment]
> when selecting radioreference as a query source it doesn't open a signin window

I have no idea what radioreference and query source is, nor signing window

**Reply by joc2** [reply]
this is what is taking about
![image](https://github.com/egzumer/uvk5-chirp-driver/assets/56229329/a9c6d646-793e-428e-a39f-f1821f077714)

but this menu is from chirp application, has i see it, you need to have user name and password to get the data from radioreference, has all other not needed.  but you should ask or report the bug on their web site.
https://chirp.danplanet.com/projects/chirp/wiki/Home.

We adapted a driver for chirp to read / write data on the computer for the specific firmware of egzumer quansheng radio uvk5.

**Reply by TheDireHavok** [reply]
Thank You, I will report to Chirp. @joc2 when I select the source radio reference I don't see the username and password part. I click on it and nothing happens.
